public/javascript/character-sheet-template.js

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Imported source code from third-party source. Reviewed and verify integrity as far as I saw at a glance.
Deeper review in the future, when we're taking full advantage of it is good idea.